<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmStausbar</class>
 <widget class="QWidget" name="frmStausbar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>170</width>
    <height>31</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnUser">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnType">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnPunctuation">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnKeyboard">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnSetup">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>btnPunctuation</zorder>
   <zorder>btnType</zorder>
   <zorder>btnUser</zorder>
   <zorder>btnKeyboard</zorder>
   <zorder>btnSetup</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
